guides/source/active_record_validations.md

```diff
@@ -258,19 +263,13 @@ returning true if any errors were found in the object, and false otherwise.
 [`invalid?`]: https://api.rubyonrails.org/classes/ActiveModel/Validations.html#method-i-invalid-3F
 [`valid?`]: https://api.rubyonrails.org/classes/ActiveRecord/Validations.html#method-i-valid-3F
 
-### `errors[]`
+### Errors
```

Comment:
Maybe we can be more descriptive here?
```suggestion
### Inspecting Errors
```

Or, as `add` is also described. Although this is maybe too ambiguous?
```suggestion
### Handling Errors
```

Reply:
I'm thinking Inspecting and Handling Errors.